Core/Time.h

```diff
@@ -236,9 +236,15 @@ static __inline __attribute__((always_inline))
 uint64_t
 rdtsc()
 {
+#if defined(__i386) || defined(__x86_64__)
```

Comment:
I actually doubt any 32-bit build works at all. It would scare me if anyone was using that. I'd rather they get the unsupported platform error. For the same reason, I think `__powerpc64__` would be better below.